### DISCUSSION TITLE
✅ Dedicated section for performance settings in Elementor

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#21194] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi elementor team.

Currently, the performance settings in Elementor settings are very separate from each other. So it would be great if they were all in one place.
For example, the option to disable Google fonts is in the advanced settings section. On the other hand, the Improved CSS Loading option is located in the Experiments section. This is not really good.

### Describe the solution you'd like

The image below is something that would be great.

![performance](https://user-images.githubusercontent.com/88034902/215485022-6b2ca6df-31e3-45f2-8267-38ac11f82c42.png)

All the options related to performance and of course stable will be in this section. All available options as well as options that will be added in the future to improve performance. **The following options will be located in this section.**

- disable Google Fonts
- Inline Font Icons
- Lazy Load Background Images
- Optimized DOM Output
- Improved Asset Loading
- Improved CSS Loading

This also makes the Experiments section cleaner than it is now.

**Comment by loucasl** [comment]
Aren't most of the experiments you listed just temporary? AKA they'll be merged into core (no longer an option) when the time comes?

**Comment by nicholaszein** [comment]
Hey @Michaelgimii!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.22.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
